encodings/datetime-parts/src/compute/compare.rs

```diff
@@ -111,6 +115,19 @@ fn compare_ne(
     Ok(Some(comparison))
 }
 
+fn compare_lt(
```

Comment:
I think all these functions are the same now?

Reply:
Yep, can just unify on one implementation here.